dateparser/date_parser.py

```diff
@@ -8,7 +9,10 @@
 from dateutil import parser
 from dateutil.relativedelta import relativedelta
 
-from timezones import pop_tz_offset_from_string, convert_to_local_tz
+from dateparser.timezones import pop_tz_offset_from_string, convert_to_local_tz
+import sys
+
+PY2 = sys.version_info[0] == 2
```

Comment:
Hi guys,
It was brought to my attention, that `dateutil` [requires six](https://github.com/dateutil/dateutil/blob/master/setup.py#L35) anyway. So what do you think about we use `six` instead of reinventing the wheel? :smiley: